lib/cisco_node_utils.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
```

Comment:
I know for internal code the tools only enforce copyright updates when touching files. Probably harmless to update across the board but IANAL.